### DISCUSSION TITLE
send an email in a certain time

### DISCUSSION THREAD

**Discussion by JUXCDRF** [PHPMailer/PHPMailer#2308] [Show and tell]
Sorry,
Recently, I am trying to speak English.
I would like to know if it is possible to configure phpmailer to send an email with a specific time to be sent like if you programmed a time to be sent an email in Gmail and it´s possible could you show me how you made?
thanks.

**Comment by Synchro** [comment]
No, PHPMailer just sends immediately. You would achieve that by choosing when to ask it to send at a higher level in your own application. In Laravel for example you might use [a delayed job](https://laravel.com/docs/8.x/queues#delayed-dispatching).

**Reply by Neoooby-Doo** [reply]
Can i use **cron job** for send mail in a certain time?

**Reply by Synchro** [reply]
Yes. That's essentially the same thing as a Laravel scheduled job, but using system-level features instead of a PHP framework.
